test/fixtures/photos.yml

```diff
@@ -16,3 +16,10 @@ two:
   height: 3
   width: 3
 
+three:
+  repository_id: 3
+  image_file_name: p2
+  image_content_type: .png
+  image_file_size: 3
+  height: 3
+  width: 3
```

Comment:
@tahnok What is a proper way of setting up fixtures for a photo? (also an array of photos)
I'm trying to see if these photos are valid. In the console I did ->
=> photo = Photo.find_by_repository_id(1)
then
=> photo.valid?
and returns false.
 I changed their file names to image.png, but that didn't solve the issue.